plugins/plugin-chart-echarts/src/Graph/controlPanel.tsx

```diff
@@ -28,22 +28,20 @@ import {
 } from '../controls';
 
 const noopControl = { name: 'noop', config: { type: '', renderTrigger: true } };
-const columnSelectControl = {
+
+const requiredEntity = {
+  ...sharedControls.entity,
+  clearable: false,
+  // do not export private name 'filterOption'
+  filterOption: sharedControls.entity,
```

Comment:
Shouldn't `filterOption` be a function? I didn't get what the comment `do not export private name 'filterOption'` means. Are you just trying to disable the type-to-search filter?

Reply:
Hi, @ktmud
typescript raise a type error `TS4082: Default export of the module has or is using private name 'FilterOption'.`, so I guess `filterOption` is a private name.

<img width="893" alt="image" src="https://user-images.githubusercontent.com/2016594/110406957-028f0100-80be-11eb-842f-ab55612faf66.png">

Reply:
thanks for this suggestion. I have changed with this suggestion. Unfortunately, The main repo also needs some repairs.